api/main_endpoints/routes/User.js

```diff
@@ -509,4 +509,27 @@ router.post('/usersSubscribedAndVerified', function(req, res) {
     });
 });
 
+// Search for all members with verified emails, subscribed, and not banned or pending
+router.post('/usersValidVerifiedAndSubscribed', function(req, res) {
+  if (!checkIfTokenSent(req)) {
+    return res.sendStatus(FORBIDDEN);
+  } else if (!checkIfTokenValid(req, membershipState.OFFICER)) {
+    return res.sendStatus(UNAUTHORIZED);
+  }
+
+  User.find({
+    emailVerified: true,
+    emailOptIn: true,
+    accessLevel: { $gte: membershipState.NON_MEMBER }
+  })
+    .then((users) => {
+      const emails = users.map(user => user.email);
+      return res.status(200).json({ emails });
+    })
+    .catch((err) => {
+      logger.error('/usersValidVerifiedAndSubscribed/ had an error:', err);
+      res.status(BAD_REQUEST).send({ message: 'Bad Request.' });
```

Comment:
by returning 400 this already means bad request, so we can just do
```suggestion
      res.sendStatus(BAD_REQUEST);
```